spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/zuul/filters/route/SimpleHostRoutingFilter.java

```diff
@@ -294,43 +297,50 @@ private HttpResponse forward(HttpClient httpclient, String verb, String uri,
 		}
 	}
 
-	protected HttpRequest buildHttpRequest (String verb, String uri, InputStreamEntity entity,
-											MultiValueMap<String, String> headers, MultiValueMap<String, String> params)
-	{
+	protected HttpRequest buildHttpRequest(String verb, String uri,
+			InputStreamEntity entity, MultiValueMap<String, String> headers,
+			MultiValueMap<String, String> params, HttpServletRequest request) {
 		HttpRequest httpRequest;
+		String uriWithQueryString = uri + (this.forceOriginalQueryStringEncoding
+				? getEncodedQueryString(request) : this.helper.getQueryString(params));
 
 		switch (verb.toUpperCase()) {
 			case "POST":
-				HttpPost httpPost = new HttpPost(uri + this.helper.getQueryString(params));
+				HttpPost httpPost = new HttpPost(uriWithQueryString);
 				httpRequest = httpPost;
 				httpPost.setEntity(entity);
 				break;
 			case "PUT":
-				HttpPut httpPut = new HttpPut(uri + this.helper.getQueryString(params));
+				HttpPut httpPut = new HttpPut(uriWithQueryString);
 				httpRequest = httpPut;
 				httpPut.setEntity(entity);
 				break;
 			case "PATCH":
-				HttpPatch httpPatch = new HttpPatch(uri + this.helper.getQueryString(params));
+				HttpPatch httpPatch = new HttpPatch(uriWithQueryString);
 				httpRequest = httpPatch;
 				httpPatch.setEntity(entity);
 				break;
 			case "DELETE":
-				BasicHttpEntityEnclosingRequest entityRequest = new BasicHttpEntityEnclosingRequest(verb,
-						uri + this.helper.getQueryString(params));
+				BasicHttpEntityEnclosingRequest entityRequest = new BasicHttpEntityEnclosingRequest(
+						verb, uriWithQueryString);
 				httpRequest = entityRequest;
 				entityRequest.setEntity(entity);
 				break;
 			default:
 				httpRequest = new BasicHttpRequest(verb,
-						uri + this.helper.getQueryString(params));
-				log.debug(uri + this.helper.getQueryString(params));
-		}
+						uriWithQueryString);
+				log.debug(uriWithQueryString);
+			}
```

Comment:
I used the formatter.xml you mention on the readme.md but I might have messed something up, I'll have a look